plugins/woocommerce/includes/abstracts/abstract-wc-session.php

```diff
@@ -130,11 +132,11 @@ public function set( $key, $value ) {
 	}
 
 	/**
-	 * Get customer ID.
+	 * Get customer ID. If the session is not initialized, returns an empty string.
 	 *
 	 * @return string
 	 */
 	public function get_customer_id() {
-		return $this->_customer_id;
+		return $this->_customer_id ?? '';
```

Comment:
This ensures the function never returns `null` which makes consuming the value slightly cleaner as we can do away with `null` checks and type casting so long as this method is used.